package.json

```diff
@@ -30,7 +30,7 @@
     "node-gyp-build": "^3.9.0",
     "p-limit": "^3.1.0",
     "pify": "^5.0.0",
-    "pprof-format": "^2.0.4",
+    "pprof-format": "^2.0.5",
```

Comment:
This will fail until the release actually happens.

Reply:
Yes, I did not expect that the release would take so much time.